concourse/tasks/build-container-image.yaml

```diff
@@ -0,0 +1,19 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/kaniko-project/executor
+    tag: debug
```

Comment:
using their exported debug tag might not be maintainable. do they publish it as regularly, and test it? we could consider our own wrapper container that tracks the upstream if not.

Reply:
From the release page, we can see debug image release with executor images

Reply:
do they test it? do they offer any guarantees it will continue to be published? etc.